bids/analysis/analysis.py

```diff
@@ -193,13 +193,21 @@ def setup(self, input_nodes=None, identity_contrasts=True, **kwargs):
                 node_coll = self._concatenate_input_nodes(input_nodes)
                 colls.append(node_coll)
 
+            model = self.model or {}
+
+            if self.level == 'run':
```

Comment:
Is there a reason to limit this to `'run'`? Right now it won't make sense to specify `"HRF_variables"` at any other level, but in principle the spec allows for estimation on, e.g., all subjects/runs at once. And `variables` could in principle be applied at other levels besides `'run'`. So I think we can replace this with a check to see if `'variables'` exists in the model.

Reply:
Presumably, `HRF_variables` requires `variables` to exist right? If so, I think I can just remove that if clause, and the rest should work (since it defaults to `[]`)